closure/goog/net/jsloader.js

```diff
@@ -111,8 +123,10 @@ goog.net.jsloader.loadMany = function(uris, opt_options) {
   if (isAnotherModuleLoading) {
     // jsloader is still loading some other scripts.
     // In order to prevent the race condition noted above, we just add
-    // these URIs to the end of the scripts' queue and return.
-    return;
+    // these URIs to the end of the scripts' queue and return the deferred
```

Comment:
I got that. But aside from the word 'deferred' (I tend to write instances in lower case but probably one can argue for upper case here), I didn't spot any case issues in other words that I added. Is it okay as amended?